modules/date/date-parser.c

```diff
@@ -161,7 +167,7 @@ date_parser_free(LogPipe *s)
 {
   DateParser *self = (DateParser *)s;
 
-  g_free(self->date_format);
+  g_list_free_full(self->date_formats, g_free);
   g_free(self->date_tz);
```

Comment:
again, a good candidate for extraction into a the string-list set of functions.

Reply:
This could be converted to string_list_free()

Reply:
Ooops, missed that one!